shippable.yml

```diff
@@ -8,7 +8,7 @@ env:
 
 build:
   pre_ci_boot:
-    image_name: shadowrobot/ubuntu-ros-indigo-build-tools
+    image_name: shadowrobot/ubuntu-ros-kinetic-build-tools
```

Comment:
where have you taken this image?

Reply:
Put the correct image and tag in the latest commit